lib/routes/films.js

```diff
@@ -0,0 +1,52 @@
+const { Router } = require('express');
+const Film = require('../models/Film');
+
+module.exports = Router()
+  .post('/', (req, res, next) => {
+    const { 
+      title,
+      studio,
+      released,
+      cast
+    } = req.body;
+
+    Film
+      .create({ title, studio, released, cast })
+      .then(film => res.send(film))
+      .catch(next);
+  })
+
+  .get('/', (req, res, next) => {
+    Film
+      .find()
+      .select({
+        title: true, 
```

Comment:
what about just .select({ __v: false, _id: false, 'cast.role': false}); it's shorter!